quickget

```diff
@@ -1023,6 +1038,30 @@ function get_opensuse() {
     make_vm_config "${ISO}"
 }
 
+function get_oraclelinux() {
+    local HASH=""
+    local ISO=""
+    local URL=""
+
+    local arch="x86_64"
+    
+    local majorver=${RELEASE::1}
+    local minorver=${RELEASE:2:1}
+  
+    local baseurl="https://yum.oracle.com/ISOS/OracleLinux/OL${majorver}/u${minorver}/${arch}/"
+    local hashurl="https://linux.oracle.com/security/gpg/checksum/OracleLinux-R${majorver}-U${minorver}-Server-x86_64.checksum"
+    
+    validate_release "releases_oraclelinux"
+
```

Comment:
I think for 6.x and 7.x  its just  inserting "-Server" before "-${arch}" in the ISO (so a case/if with `major<8 then` logic and a local to hold the string (or not)
Mind you - I'd not bother with 6.x or 7.x myself, and be suprised if anyone wants <8.4 at this stage so the above isn't worth bothering with.  Obv. too much wittering so must be time for bed.

Reply:
> I think for 6.x and 7.x its just inserting "-Server" before "-${arch}" in the ISO (so a case/if with `major<8 then` logic and a local to hold the string (or not) Mind you - I'd not bother with 6.x or 7.x myself, and be suprised if anyone wants <8.4 at this stage so the above isn't worth bothering with. Obv. too much wittering so must be time for bed.

Good catch! I added 7.9, 7.8 and 7.7 support. I agree it doesn't make sense to add 6.x at this point.